features/comment.feature

```diff
@@ -2,7 +2,8 @@ Feature: Manage WordPress comments through the REST API
 
   Background:
     Given a WP install
-    And I run `wp plugin install rest-api --activate`
+    When I run `wp core version`
+	Then STDOUT should be a version string >= 4.7
```

Comment:
> Since this only makes the test cases stronger, I can't understand why you would not allow this to remain. It is not enough to just have a WP install. It must be a WP 4.7+ install.

This specific comment isn't a strongly held opinion. However, I must point you to my previous comment on this pull request:

> At this point, I'm not confident in accepting the entire pull request. Some changes make sense, and others need more consideration. I'd like for it to be broken into logical subparts, so that each piece can be evaluated on its own.